app/src/main/res/values/strings.xml

```diff
@@ -944,7 +944,7 @@ The new line here must be kept as the second half of the string is clickable for
 
     <!-- Contextual Feature Recommendation Popups -->
     <!-- CFR for Toolbar Shield Icon (Banner Info Message) -->
-    <string name="cfr_for_toolbar_shield_icon">Looks like this site was trying to track you. Click the shield for site settings and tracker blocking stats.</string>
+    <string name="cfr_for_toolbar_shield_icon">Got ‘em! We stopped this site from spying on you. Tap the shield any time to see what we’re blocking.</string>
```

Comment:
Is it OK to update directly this string instead of tagging it for removal, It looks we are following the same string [removal process similar](https://github.com/mozilla-mobile/focus-android/blob/797fa7cb520eae156bfb854a106521a22b72a13d/app/src/main/res/values/strings.xml#L739) to the [Fenix](https://github.com/mozilla-mobile/fenix/wiki/Removing-strings) one.

Reply:
Good point! My bad, I forgot about the process